tests/test_path3d.h

```diff
@@ -0,0 +1,85 @@
+/*************************************************************************/
+/*  test_path_follow_3d.h                                                */
```

Comment:
```suggestion
/*  test_path_3d.h                                                */
```

The file should also be renamed to `test_path_3d.h`.